### ISSUE TITLE
Stuck when entering devtools via F12

### ISSUE THREAD

**Issue by HerbertHe** [slidevjs/slidev#193]
**Describe the bug**
Stuck when entering the devtools via F12, especially for Elements (do nothing, but waiting)

![edgebug](https://user-images.githubusercontent.com/30955264/119986281-3b23e400-bff6-11eb-8d09-69c80b94fc79.gif)

**To Reproduce**

1. `npm init slidev@latest`
2. Add following contents

````markdown
---

# 研究现状

Research Status

---

# 论文结构

Paper Structure

---

# 波的叠加原理

Superposition Principle of Wave

---

# 杨氏双缝干涉实验

Young's Double-slit Interference Experiment

---

# 杨氏双缝干涉实验仿真

Young's Double-slit Interference Experiment Simulation

---

```matlab
I_0 = 1;
for i = 1:Fineness
    r1 = sqrt((yAxis(i) - d / 2).^2 + D^2);
    r2 = sqrt((yAxis(i) + d / 2).^2 + D^2);
    B(i, :) = 4 * I_0 * cos(pi * (r2 - r1) / Lambda).^2;
end
ColorMax = 255;
ColorLevels = (B / 4.0) * ColorMax;
subplot(1, 2, 1);
image(xAxis, yAxis, ColorLevels);
colormap(gray(ColorMax));
subplot(1, 2, 2);
plot(B(:), yAxis);
```

---

# 牛顿环干涉实验

Newton's Ring Interference Experiment

---

# 牛顿环干涉实验仿真

Newton's Ring Interference Experiment Simulation

---

# 单狭缝夫琅禾费衍射实验

Single-slit Fraunhofer Diffraction Experiment

---

# 单狭缝夫琅禾费衍射实验仿真

Single-slit Fraunhofer Diffraction Experiment Simulation

---

# 光的偏振

Polarization of Light

---

# 光的偏振态实验仿真

Polarization State of Light Simulation

---

# 结论

Conclusion

---

# 引用文献

References

---

# 致谢

Thanks

````

3. preview.

change utilities of windicss, waiting about 1~2 s

Reload the page, it took about 20 s, task manager showed high CPU usage.

![bubug](https://user-images.githubusercontent.com/30955264/119987457-a28e6380-bff7-11eb-8fa4-a61f3b2cda5e.gif)

**Desktop (please complete the following information):**
 - OS: Windows10 Home Edition
 - Browser: Microsoft Edge (Chromium)
 - Slidev version: v0.15.0

**Comment by HerbertHe** [comment]
About Computer:

- Edge: 90.0.818.66 
- Windows:  Home Edition 20H2
- CPU: Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz   2.81 GHz
- RAM: 8GB + 8GB

**Comment by HerbertHe** [comment]
For reloading, it seems caused by `widgets.js`

![image](https://user-images.githubusercontent.com/30955264/119990753-4b8a8d80-bffb-11eb-8ebb-3a285f5533f8.png)

**Comment by antfu** [comment]
- Have you tried with latest Chrome? I remember there is a bug in Chrome devtools last year, I am not sure if that's the version Edge is using.
- Can you try start the server with `DEBUG=vite-plugin-windicss:scan:* npm run dev` and share me the output?
- Twitter script should only be enabled when `<Tweet/>` is used, is that true in your case?

**Comment by HerbertHe** [comment]
- About the twitter script, I noticed that it was imported by the `index.html` file from default template. When I deleted it, it reloaded quickly! And I haven't use `<Tweet />`.
- I tired with the latest Chrome, and it worked well. And the latest Edge (91.0.864.37) seems that working well too.

**Comment by HerbertHe** [comment]
the output as followed:

```text
vite-plugin-windicss:scan:glob include [
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/**/*.{vue,ts,tsx,js,jsx,md}',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/**/*.{vue,ts}',       
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/theme-seriph/setup/**/*.vue',
  '!C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules'
] +4s
  vite-plugin-windicss:scan:glob exclude [
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/.git',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/dist',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/.git',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/dist',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/node_modules',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/windi.config.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/theme-seriph/setup/node_modules',
  'C:\\Users\\Herbert\\Desktop\\毕业论文\\论文答辩\\ppt\\node_modules\\monaco-editor',
  'C:\\Users\\Herbert\\Desktop\\毕业论文\\论文答辩\\ppt\\node_modules\\katex'
] +1ms
  vite-plugin-windicss:scan:glob files [
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/README.md',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Abstract.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Author.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Background1.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Background2.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/CommonText.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Picture.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/References.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/ResearchStatus.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/ResearchStatus1.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Significance.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Youngs.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/App.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Arrow.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/AutoFitText.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/CodeHighlightController.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Mermaid.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Monaco.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/SlideCurrentNo.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/SlidesTotal.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Transform.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Tweet.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/VAfter.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/VClick.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/VClicks.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Youtube.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/env.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/iframes/monaco/index.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Controls.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/DevicesList.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Editor.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Goto.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/InfoDialog.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/MenuButton.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Modal.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/NavControls.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/NoteEditor.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Play.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Presenter.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/RecordingControls.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/RecordingDialog.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SelectList.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Settings.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlideContainer.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlideWrapper.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlidesOverview.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlidesShow.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/WebCamera.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/types.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layoutHelper.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/404.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/center.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/cover.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/default.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/end.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/fact.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/full.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/image-left.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/image-right.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/image.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/intro.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/none.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/quote.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/section.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/statement.vue',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/dark.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/nav.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/note.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/recording.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/route.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/shortcuts.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/main.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/modules/context.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/modules/directives.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/modules/mermaid.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/routes.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/codemirror.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/main.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/mermaid.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/monaco.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/prettier.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/root.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/shim.d.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/state/index.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/utils.ts',
  'C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/slides.md'
] +121ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/README.md +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Abstract.vue +47ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Author.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Background1.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Background2.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/CommonText.vue +61ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Picture.vue +153ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/References.vue +10ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/ResearchStatus.vue +67ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/ResearchStatus1.vue +83ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Significance.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/components/Youngs.vue +70ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/App.vue +144ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Arrow.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/AutoFitText.vue +1ms    
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/CodeHighlightController.vue +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Mermaid.vue +153ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Monaco.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/SlideCurrentNo.vue +2ms 
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/SlidesTotal.vue +0ms    
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Transform.vue +1ms      
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Tweet.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/VAfter.ts +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/VClick.ts +84ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/VClicks.ts +215ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/builtin/Youtube.vue +0ms        
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/env.ts +3ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/iframes/monaco/index.ts +106ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Controls.vue +203ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/DevicesList.vue +1ms  
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Editor.vue +1ms       
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Goto.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/InfoDialog.vue +2ms   
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/MenuButton.vue +1ms   
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Modal.vue +91ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/NavControls.vue +1ms  
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/NoteEditor.vue +1ms   
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Play.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Presenter.vue +0ms    
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/RecordingControls.vue 
+2ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/RecordingDialog.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SelectList.vue +2ms   
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/Settings.vue +0ms     
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlideContainer.vue +97ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlideWrapper.vue +3ms 
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlidesOverview.vue +107ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/SlidesShow.vue +93ms  
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/WebCamera.vue +183ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/internals/types.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layoutHelper.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/404.vue +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/center.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/cover.vue +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/default.vue +1ms        
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/end.vue +2ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/fact.vue +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/full.vue +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/image-left.vue +1ms     
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/image-right.vue +89ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/image.vue +187ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/intro.vue +201ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/none.vue +110ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/quote.vue +3ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/section.vue +110ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/layouts/statement.vue +3ms      
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/dark.ts +118ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/nav.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/note.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/recording.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/route.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/logic/shortcuts.ts +90ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/main.ts +208ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/modules/context.ts +3ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/modules/directives.ts +0ms      
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/modules/mermaid.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/routes.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/codemirror.ts +0ms        
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/main.ts +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/mermaid.ts +2ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/monaco.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/prettier.ts +0ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/setup/root.ts +96ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/shim.d.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/state/index.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/node_modules/@slidev/client/utils.ts +1ms
  vite-plugin-windicss:scan:transform C:/Users/Herbert/Desktop/毕业论文/论文答辩/ppt/slides.md +1ms

```

**Comment by antfu** [comment]
Ah, do you have `index.html` in your project? Guess it's generate from a failed building (it doesn't get removed). If you don't have custom content in it, I think you can safely remove it.

**Comment by antfu** [comment]
Found a style duplication bug in vite-plugin-windicss. Can you upgrade to `v0.19.2` to see if it's a fix for you? Thanks

**Comment by HerbertHe** [comment]
Sorry, I am busy for graduation thesis recently. The bug has been fixed at `v0.19.5`, it works very well!

I found that the `favicon.png` was loaded from `raw.githubusercontent.com` by default, I thought it could be loaded from `jsdelivr` for shorter time.

Maybe it can be replaced as `https://cdn.jsdelivr.net/gh/slidevjs/slidev@main/assets/favicon.png`

**Comment by antfu** [comment]
Yeah, good idea. We could change all rawcontents to jsdilver. Mind send a PR for it? (no hurries)

Close as the original issue get solved.
